### DISCUSSION TITLE
[2일차-지원] Put과 Patch의 차이점

### DISCUSSION THREAD

**Discussion by smarfy99** [zzangCS/zzangInterview#9] [프론트 면접 질문]
HTTP요청 중 Put과 Patch의 차이점과 각각의 기능에 대해 설명해보세요.

**Comment by smarfy99** [comment]
put 메서드는 자원을 교체하는 HTTP 요청이다. 클라이언트가 해당 자원의 상태를 모두 알고 있어야 한다.
반면 patch 메서드는 자원을 부분적으로 수정하기 위한 HTTP 요청이다.

**Comment by skyhanull** [comment]
put과 patch 둘다 수정을 하는 http메소드이지만
put은 전체를 수정할 때 사용하고
patch는 일부를 수정할 때 사용한다.

**Comment by ez1n** [comment]
put 메소드 : 요청 페이로드를 사용하여 새로운 리소스를 생성하거나, 대상 리소스를 나타내는 데이터를 대체함 (리소스의 전체를 업데이트)

patch 메소드 : 리소스의 일부를 업데이트한다. (부분적인 수정)
